Vagrantfile

```diff
@@ -488,6 +488,14 @@ Vagrant.configure("2") do |config|
           libvirt.nested = true
           libvirt.default_prefix = ''
           libvirt.management_network_address = libvirt_management_network_address
+
+          if($additional_disk_size > 0 && (!hostname.include? $base_box_builder_vm_name))
+              libvirt.storage :file,
+                :size => $additional_disk_size,
+                :path => hostname + '_sdb.img',
```

Comment:
Quick question: where does the provider creates the img file?

Reply:
in the same storage pool as the VMs primary disk, generally in the default path used by libvirt